StreamLit.md

```diff
@@ -0,0 +1,201 @@
+# StreamLit
```

Comment:
Please add the cheatsheet metadata, you can refer to other cheatsheets for the format

Reply:
Hi @onecompiler-ops I have added all the metadata and the proper format. Please check.